comment/models/blocker.py

```diff
@@ -0,0 +1,30 @@
+from django.contrib.auth import get_user_model
+from django.db import models
+
+from comment.messages import BlockState
+from comment.managers import BlockedUserManager
+
+
+class BlockedUser(models.Model):
+    user = models.ForeignKey(get_user_model(), on_delete=models.CASCADE, blank=True, null=True)
+    email = models.EmailField(blank=True, null=True)
+    blocked = models.BooleanField(default=True)
+
+    objects = BlockedUserManager()
+
+    def __str__(self):
+        return getattr(self.user, self.user.USERNAME_FIELD) if self.user else self.email
```

Comment:
would this function be helpful here: https://github.com/Radi85/Comment/blob/develop/comment/models/comments.py#L86 ?

Reply:
I need full email here unlike `get_username` method.